src/main/java/nextstep/subway/member/application/MemberService.java

```diff
@@ -33,5 +39,7 @@ public void updateMember(Long id, MemberRequest param) {
 
     public void deleteMember(Long id) {
         memberRepository.deleteById(id);
+
+        fileLogger.info("[member create] id - {}", id);
```

Comment:
```suggestion
        fileLogger.info("[member delete] id - {}", id);
```

회원가입, 탈퇴시 로그도 꼼꼼히 잘 남겨주셨네요 👍